From 6e68171d2898ea59bbf71cb4bfa661dcfd42899c Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:10:45 -0400
Subject: [usb] pl2303: remove redundant line-request call

Message-id: <1410887463-20674-574-git-send-email-dzickus@redhat.com>
Patchwork-id: 95362
O-Subject: [RHEL7 PATCH 573/591] USB: pl2303: remove redundant line-request call
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 294bb2c6178da6334f54bd88aac7dca63fd89b06
Author: Johan Hovold <jhovold@gmail.com>
Date:   Sun Dec 29 19:23:04 2013 +0100

    USB: pl2303: remove redundant line-request call

    Remove redundant get_line_request (the read back settings are never
    used).

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/pl2303.c b/drivers/usb/serial/pl2303.c
index 80a9ab1..d3a1505 100644
--- a/drivers/usb/serial/pl2303.c
+++ b/drivers/usb/serial/pl2303.c
@@ -518,9 +518,6 @@ static void pl2303_set_termios(struct tty_struct *tty,
   spin_unlock_irqrestore(&priv->lock, flags);
  }
 
- memset(buf, 0, 7);
- pl2303_get_line_request(port, buf);
-
  if (C_CRTSCTS(tty)) {
   if (spriv->type == HX)
    pl2303_vendor_write(serial, 0x0, 0x61);
-- 
1.7.1